package.json

```diff
@@ -60,6 +59,7 @@
   "scripts": {
     "test": "nyc mocha test --file test/logfile.setup.js --recursive --exit",
     "posttest": "eslint ./ && license-checker --summary && ./node_modules/lockfile-lint/bin/lockfile-lint.js --path package-lock.json —type npm --allowed-hosts npm artifactory.corp.adobe.com --allowed-schemes \"https:\" \"file:\" --empty-hostname true",
-    "beautify": "eslint ./ --fix"
+    "beautify": "eslint ./ --fix",
+    "unit-test": "nyc mocha test --file test/logfile.setup.js --recursive --exit"
```

Comment:
This is exactly the same as `test`?

Reply:
Nearly. There is no eslint or license check running after this one.

Reply:
(I'll rework those scripts in another PR anyway).

Reply:
Then to achieve DRY you could do:

```
"test": "npm run unit-test",
"unit-test": "nyc mocha ...."
```